From f627f0625914d8a79e7023d742e738378273d269 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Mon, 16 Apr 2012 21:18:48 -0300
Subject: [PATCH 25/40] 4.4.1 Cuenca de Sholazar: Atormentar a los
 badanudillos

Mejoras en la jugabilidad de la quest "Tormenting the Softknuckles"
---
 ...04_16_101_world_tormenting_the_softknuckles.sql |   13 +++++++++++++
 1 files changed, 13 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_16_101_world_tormenting_the_softknuckles.sql

diff --git a/sql/updates/world/reanemu/2012_04_16_101_world_tormenting_the_softknuckles.sql b/sql/updates/world/reanemu/2012_04_16_101_world_tormenting_the_softknuckles.sql
new file mode 100644
index 0000000..114da04
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_16_101_world_tormenting_the_softknuckles.sql
@@ -0,0 +1,13 @@
+-- Tormenting the Softknuckles - Quest 12530
+SET @MONITO :=28127;
+SET @MATRIARCH :=28213;
+
+DELETE FROM `creature` WHERE `id`=@MATRIARCH;
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry` IN (@MONITO,@MATRIARCH);
+DELETE FROM `creature_ai_scripts` WHERE `creature_id`=@MATRIARCH;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (@MONITO,@MATRIARCH) AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@MONITO,0,0,0,8,0,30,1,50979,0,0,0,12,@MATRIARCH,6,5000,0,0,0,8,0,0,0,5121.923340,3982.503662,-61.803101,1.123347,'Softknuckle - Spellhit - chance to summon Hardknuckle Matriarch'),
+(@MATRIARCH,0,0,0,4,0,100,0,0,0,0,0,11,54287,0,0,0,0,0,1,0,0,0,0,0,0,0,'Hardknuckle Matriarch - Aggro - cast Enrage'),
+(@MATRIARCH,0,1,0,0,0,100,0,6000,10000,12000,15000,11,61580,0,0,0,0,0,1,0,0,0,0,0,0,0,'Hardknuckle Matriarch - In combat - Cast Thunderstomp');
\ No newline at end of file
-- 
1.7.8.msysgit.0

